src/uct/ib/Makefile.am

```diff
@@ -145,6 +145,18 @@ endif # HAVE_MLX5_HW_UD
 
 endif # HAVE_TL_UD
 
+
+if HAVE_EFA_DV
+libuct_ib_la_LDFLAGS +=  $(LIB_EFA)
```

Comment:
LIB_EFA -> EFA_LDFLAGS

Reply:
I actually don't add any extra LDFLAGS for EFA, it is the library name itself. I get your point that the library should not be added to LDFLAGS. But, that's what we have done for NUMA_LIBS and LIB_MLX5 in this same file. So, I'm a bit confused.